### DISCUSSION TITLE
Найцікавіші новини місяця: грудень 2023

### DISCUSSION THREAD

**Discussion by rkiyanchuk** [shopokodu/community#43] [Теми для подкасту]
**Теми ведучих:**

- [ ] Бути [BSOD на Linux](https://www.phoronix.com/news/systemd-255).
- [ ] Більше не буде замочків в браузерах.
![CleanShot 2023-12-15 at 22 47 57@2x](https://github.com/shopokodu/community/assets/843725/69960abe-cfe8-47ef-9a94-8ea6191a46b6)

**Теми случахів:**

- [ ]

**Comment by vuwnevska** [comment]
Кібер атака на Київстар, ваші версії що вони поклали чи погоджуєтесь що просто видаляли все підряд
https://x.com/loony_dev/status/1735249944603250955?s=20
